i/istack-commons/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "maintainer": "mayurbhosure",
+  "package_name": "istack-commons",
```

Comment:
package name should be jaxb-istack-commons and accordingly update folder name, script name etc.